src/tiledb/cloud/utilities/common.py

```diff
@@ -0,0 +1,197 @@
+import configparser
+import logging
+import os
+import sys
+from typing import Any, Mapping
+
+import tiledb
+import tiledb.cloud
+
+
+def read_aws_config(
```

Comment:
Does this need to be exported for general use? It seems like it is mostly useful as an internal tool for other functions.

It's fine if we want to; I just want to make sure we keep a sensible API surface.

Reply:
This is useful in a notebook environment, where we have to read the credentials from somewhere. We have been passing around functions in notebooks that do the same thing, so I thought we could formalize it.

It's fine if we take some time to think about it. Let me know and I can remove `read_aws_config`.